osquery/killswitch/plugins/killswitch_filesystem.h

```diff
@@ -0,0 +1,20 @@
+#include "osquery/killswitch/plugins/killswitch_json.h"
+#include <string>
+
+namespace osquery {
+
+class KillswitchFilesystem : public KillswitchJSON {
+ public:
+  KillswitchFilesystem();
+  KillswitchFilesystem(const std::string& conf_path);
+
+ protected:
+  virtual Status getJSON(std::string& content) override;
+
+ private:
+  std::string conf_path_;
```

Comment:
1. Would you like to use a `boost::filesystem::path` class here instead of `std::string`?
2. Do you need to change this path during after creation? May be it is a good place for `const`?